### PR TITLE
LPS-181232 | Refactor tests that have macros with only 1 assert

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-181232](https://issues.liferay.com/browse/LPS-181232)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:relevant

**Comment by davisaints** [comment]
ci:test:object

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 1]
```diff
@@ -146,7 +146,7 @@ definition {
 
 			FDSCustomView.selectView(key_itemName = "TEST");
 
-			ObjectCustomViews.assertColumnNotPresent(columnName = "Description");
+			FDSCustomView.assertColumnNotPresent(columnName = "Description");
```
Comment (timpak) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:149]: These use different paths, can you check if this works correctly?

**Review by timpak — COMMENTED** [review]
@davisaints can you analyze the test results and check every test you touched? Great job with this btw. If the tests are passing, you can forward this.

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase (hunk 1 above)
Comment (davisaints) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:149]: Yes, they are different paths. I'll fix that.

**Comment by davisaints** [comment]
Hey @timpak, there was just one related issue that has now been fixed. I will wait for these unrelated source format errors to be fixed to forward. Thanks for reviewing!

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:forward

**Pull request closed without merging**